### PR TITLE
Remove YUI from topo events

### PR DESCRIPTION
**Pull request by huwshimi**
In this branch:
- update hoverService events to use addEventListener
- remove unused rescaled, addRelationEnd and resized events

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add a couple of apps
- hovering an app should highlight it in the list of services

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7258/

File: jujugui/static/gui/src/app/views/topology/relation.js [hunk 1]
```diff
@@ -677,8 +677,6 @@ YUI.add('juju-topology-relation', function(Y) {
         // TODO clean up, abstract
         self.cancelRelationBuild();
       }
-      // Signify that the relation drawing has ended.
-      topo.fire('addRelationEnd');
```
Comment (frankban) [inline jujugui/static/gui/src/app/views/topology/relation.js]: Are those events (addRelationEnd and resized) unused?

**Comment by jaycee** [comment]
QA is :Ok:

File: jujugui/static/gui/src/app/views/topology/relation.js (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/views/topology/relation.js]: That's correct! (Sorry, I did leave a note in the PR description, but it's not really obvious with all the comments/label updates etc.)

**Comment by huwshimi** [comment]
Thanks @jaycee @frankban :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**